addon/addon/models/orchestration.ts

```diff
@@ -205,3 +171,29 @@ export class OrchestrationMatrix {
     return new OrchestrationMatrix(rows, maxLength);
   }
 }
+
+
+export interface AnimationDefinition {
+  timeline: AnimationTimeline;
+}
+
+export type AnimationTimeline = {
+  type: 'sequence' | 'parallel';
+  animations: (MotionDefinition | AnimationTimeline)[];
+};
+
+export interface MotionDefinition {
+  sprites: Set<Sprite>;
+  properties: {
+    [k in MotionProperty]?: MotionOptions | Record<string, never>;
```

Comment:
Issue was not making these motion properties optional. Could also make it so that we need at least one motion property, but I think that can come later, if we need it?